### ISSUE TITLE
metadata rendering - dataset annotations

### ISSUE THREAD

**Issue by mobb** [sbclter/sbclter-website-2019#77]
these are a new feature we want to support.

they show up in 2 places in the EML,
dataset/annotation
dataset/dataTable/attributeList/attribute/annotation

dataset's with annotations:
TO BE ADDED BY LI KUI

**Comment by lkuiucsb** [comment]
dataset with annotations: knb-lter-sbc.127 and knb-lter-sbc.57

**Comment by mobb** [comment]
example:
https://portal.edirepository.org/nis/metadataviewer?packageid=knb-lter-sbc.127.1

<img width="496" alt="Screen Shot 2020-05-13 at 2 18 45 PM" src="https://user-images.githubusercontent.com/4931977/81866944-d13cb780-9524-11ea-8540-7a78a60ae797.png">

**Comment by lkuiucsb** [comment]
@lleeoochen please add the hyperlink to the text Carbon Percentage and Nitrogen Percentage. So people can click into it and go to the dictionary page

**Comment by mobb** [comment]
Agreed - For annotations the valueURI field should become the `href=""` in an anchor tag. 
But not for the property URI, ie, Do not construct an anchor tag for this, al we want is the label (as is)

Example is from data table 2, columns labeled PERENT_CARBON, PERCENT_NITROGEN.
